cmd/otel-allocator/main.go

```diff
@@ -152,6 +154,7 @@ func newServer(log logr.Logger, allocator allocation.Allocator, discoveryManager
 	}
 	router := mux.NewRouter().UseEncodedPath()
 	router.Use(s.PrometheusMiddleware)
+	router.HandleFunc("/scrape_configs", s.ScrapeConfigsHandler).Methods("GET")
```

Comment:
I'm worried about the response sizes and traffic which would be generated by this. Every collector would request all scrapes_configs on every sync tic. 

Can you post an example response? 

I think there should be parameter as it is implemented at the targets endpoint https://github.com/open-telemetry/opentelemetry-operator/pull/1124/files#diff-3ea793544213bc190a7701f8934a02c85faa78051509e02f2ec0c4ff3fcef608R237

Reply:
K, then solve issues as they arise and not block this moving forward.